src/pages/StudentLogin/StudentLogin.js

```diff
@@ -17,16 +17,18 @@ const StudentLogin = (props) => {
       }
     )
   }
-
   const handleSubmit = (e) => {
+    const handleUserLoginObject={field}
```

Comment:
It looks like handleUserLoginObject only exists to pass the data down to verifyLogin
I think it makes sense to just pass the value directly in the verifyLogin and save a step.

Reply:
Removed handleUserLoginObject